src/components/routes/ViewConfig.tsx

```diff
@@ -50,6 +69,14 @@ const ViewConfig: SFC<FullProps> = props => {
     );
 };
 
+const submitViewConfig = gql`
```

Comment:
wouldn't it make more sense to put the mutation in `QUERY_COLLECTION_PROPERTIES` ?